srcpkgs/splatmoji/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'splatmoji'
+pkgname=splatmoji
+version=1.2.0
+revision=1
+conf_files="/etc/xdg/${pkgname}/${pkgname}.config"
+hostmakedepends="pandoc"
+depends="bash>4.4 rofi xdotool xsel jq"
+checkdepends="jq shunit2"
+short_desc="Emoji and kaomoji selector"
+maintainer="Allen Sobot <chilledfrogs@disroot.org>"
+license="MIT"
+homepage="https://github.com/cspeterson/splatmoji"
+distfiles="https://github.com/cspeterson/${pkgname}/archive/v${version}.tar.gz"
+checksum=cfcc9949ae053be2106fb64cbd75adfff5a772a3b4bc2ae15ad44e2997a0e6ee
+
+do_check() {
+	./test/*
+}
+
+pre_install() {
+	pandoc -s -t man README.md -o ${pkgname}.1
+}
+
+do_install() {
+	vbin splatmoji
+	vman ${pkgname}.1
+	vmkdir usr/lib/${pkgname}
+	vcopy lib/functions usr/lib/${pkgname}
+	vmkdir etc/xdg/${pkgname}
+	vcopy ${pkgname}.config etc/xdg/${pkgname}
+	vlicense LICENSE
+}
+
+splatmoji-emojidata_package() {
```

Comment:
Does this need to be split?

Reply:
It could theoretically be used separately iirc, based on the package documentation, so idk

Reply:
can splatmoji work without it? If not, it needs to be added as a dependency.

Reply:
It can, actually

Reply:
Umm, @abenson, so do we keep it separate or not? To me it makes the most sense separately, been using it locally as such, but hey

Reply:
> It can, actually, I checked, or I would have already

    Cannot find any data files with the provided options.. 

At least `splatmoji` should have `splatmoji-emojidata>=${version}_${revision}` listed in `depends`.  I don't think we have any other compatible data files in the repos.

Also, are there any other applications that use `splatmoji`'s data files?  If not, I don't see the point to the split.

> been using it locally as such

If you have both installed, sure, it works fine.  Right now if you just install `splatmoji` it doesn't work.

Reply:
Well, for the moment we indeed don't have any compatible ones *in the repos*, but it seems that the developer made it to work with arbitrary data files (so they could be in your homedir somewhere for instance, if you for whatever reason wanted to configure custom ones), so I would kinda prefer to have it as a separate package, but at least for now a dependency could work

Reply:
don't split, not worth it
```
du -sch *xbps
12K	splatmoji-1.2.0_1.x86_64.xbps
3.5M	splatmoji-emojidata-1.2.0_1.x86_64.xbps
```

Reply:
Fair enough

Reply:
Done